src/App.js

```diff
@@ -1,3 +1,21 @@
+
+import LocationsList from "./components/LocationsList";
+import SearchBar from "./components/SearchBar";
+import LocationForm from "./components/LocationForm";
+
+function App() {
```

Comment:
Why are you redefining and reexporting the app component which is also defined on line 22?

Reply:
Also prettier is not applied here. Read the comment I left on the next file on how to apply it more naturally and easily.